src/blech32.cpp

```diff
@@ -48,12 +41,13 @@ uint64_t PolyMod(const data& v)
     // 1*x^5 + v0*x^4 + v1*x^3 + v2*x^2 + v3*x + v4. The implicit 1 guarantees that
     // [v0,v1,v2,...] has a distinct checksum from [0,v0,v1,v2,...].
 
-    // The output is a 30-bit integer whose 5-bit groups are the coefficients of the remainder of
+    // The output is a 60-bit integer whose 5-bit groups are the coefficients of the remainder of
     // v(x) mod g(x), where g(x) is the Blech32 generator,
-    // x^6 + {29}x^5 + {22}x^4 + {20}x^3 + {21}x^2 + {29}x + {18}. g(x) is chosen in such a way
-    // that the resulting code is a BCH code, guaranteeing detection of up to 3 errors within a
-    // window of 1023 characters. Among the various possible BCH codes, one was selected to in
-    // fact guarantee detection of up to 4 errors within a window of 89 characters.
+    // x^12 + {31}x^10 + {10}x^9 + {18}x^8 + {31}x^7 + {14}x^6
+    //      +  {18}x^5 + {23}x^3 + {22}x^2 +  {4}x   +  {6}   . g(x) is chosen in such a way
+    // that the resulting code is a BCH code, guaranteeing detection of up to ??? errors within a
```

Comment:
@apoelstra Can you lookup the values needed for the 4 `???` values on lines 48 through 50?

Reply:
cc @sipa @instagibbs do either of you remember the paramaters of this code?

Reply:
Sipa would be the one to know what and why

Reply:
Given the size I think it was just algebraically constructed, not optimized for by exhaustive analysing its actual behavior. Let me run some tests to see if I can reconstruct it.